src/utmpx.rs

```diff
@@ -159,14 +170,14 @@ impl<'a> Iterator for UtmpIter<'a> {
 /// Utmp accounting database.
 #[derive(Clone, PartialEq, Eq, Debug)]
 pub struct Utmp {
-    path: CString,
+    _private: (),
```

Comment:
I'm using this just to prevent consumer from directly building this type. The `open` contructor has proper additional logic for rewinding on open. Any alternative suggestion?

Reply:
I'm using this just to prevent consumer from directly building this type. The `open` contructor has proper additional logic for rewinding on open. Any alternative suggestion?

Reply:
That makes sense.  Could you please add a comment to that effect?